### DISCUSSION TITLE
v5.0.3 release announcement has wrong link

### DISCUSSION THREAD

**Discussion by usatosi** [fluent/fluentd#4552] [General]
The blog post [fluent\-package v5\.0\.3 has been released \| Fluentd](https://www.fluentd.org/blog/fluent-package-v5.0.3-has-been-released) seems to have wrong link to http://localhost:9395/blog/upgrade-td-agent-v4-to-v5 with text "[Upgrade to fluent-package v5](http://localhost:9395/blog/upgrade-td-agent-v4-to-v5)".

Please guide me to suitable location if here is not the place to file such issue.

**Comment by daipom** [comment]
Thanks for your report!
I have made a PR: https://github.com/fluent/fluentd-website/pull/316
This will be fixed soon.

You can report issues about the Fluentd website to https://github.com/fluent/fluentd-website/issues!
